hugegraph-core/src/test/java/com/baidu/hugegraph/util/StringUtilTest.java

```diff
@@ -0,0 +1,34 @@
+package com.baidu.hugegraph.util;
+
+import org.junit.Test;
+import java.util.ArrayList;
+
+import static org.junit.Assert.*;
+import static com.baidu.hugegraph.util.StringUtil.*;
+
+public class StringUtilTest {
+
+    @Test
+    public void testDesc() {
+        assertEquals(".\'\'()", desc("\'\'", new ArrayList<>()));
+        assertEquals(".1\'2\'3()", desc("1\'2\'3", new ArrayList<>()));
+        assertEquals(".a\'b\'c()", desc("a\'b\'c", new ArrayList<>()));
+    }
+
+    @Test
+    public void testEscape() {
+        assertEquals("\u000011213", escape('1', '\u0000', new String[]{"1", "2", "3"}));
+        assertEquals("a1b1c", escape('1', '\u0000', new String[]{"a", "b", "c"}));
+    }
+
+    @Test
+    public void testUnescape() {
+        assertEquals(1, unescape("", "", "").length);
+        assertEquals(1, unescape("foo", "bar", "baz").length);
+    }
+
+    @Test
+    public void testValueOf() {
+        assertEquals(12, valueOf(Integer.class, "12"));
```

Comment:
also add cases for Byte,Short,Long,Float,Double

Reply:
additional test cases added